db/db.js

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from 'mongodb'
+
+let uri = process.env.MONGODB_URI
+let dbName = process.env.MONGODB_DB
+
+let cachedClient = null
+let cachedDb = null
+
+if (!uri) {
+  throw new Error(
+    'Please define the MONGODB_URI environment variable inside .env.local'
+  )
+}
+
+if (!dbName) {
+  throw new Error(
+    'Please define the MONGODB_DB environment variable inside .env.local'
+  )
+}
+
+export async function connectToDatabase() {
+  if (cachedClient && cachedDb) {
+    return { client: cachedClient, db: cachedDb }
+  }
+
+  const client = await MongoClient.connect(uri, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+
+  }
+  )
+  console.log("connected")
```

Comment:
Like the message over here is not clear connected to what? so mention it connect to database <database-name>